Java/core/src/main/java/com/amazon/randomcutforest/tree/Node.java

```diff
@@ -480,11 +490,12 @@ BoundingBox constructBoxInPlace(BoundingBox currentBox) {
         }
     }
 
-    void reComputePointSum(double[] point) {
+    void reComputePointSum() {
```

Comment:
Let's write `recompute` instead of `reCompute` to be consistent.

Reply:
Done.